services/brig/test/integration/API/User/Account.hs

```diff
@@ -141,9 +141,57 @@ tests _ at opts p b c ch g aws =
       testGroup
         "temporary customer extensions"
         [ test' aws p "domains blocked for registration" $ testDomainsBlockedForRegistration opts b
+        ],
+      testGroup
+        "update user email by team owner"
+        [ test' aws p "put /users/:uid/email" $ testUpdateUserEmailByTeamOwner b
         ]
     ]
 
+testUpdateUserEmailByTeamOwner :: Brig -> Http ()
+testUpdateUserEmailByTeamOwner brig = do
+  (_, teamOwner, emailOwner : otherTeamMember : _) <- createPopulatedBindingTeamWithNamesAndHandles brig 2
+  (teamOwnerDifferentTeam, _) <- createUserWithTeam' brig
+  newEmail <- randomEmail
+  initiateEmailUpdateNoSend brig newEmail (userId emailOwner) !!! (const 202 === statusCode)
+  checkActivationCode newEmail True
+  checkLetActivationExpire newEmail
+  checkActivationCode newEmail False
+  checkSetUserEmail teamOwner emailOwner newEmail 200
+  checkActivationCode newEmail True
+  checkUnauthorizedRequests emailOwner otherTeamMember teamOwnerDifferentTeam newEmail
+  activateEmail brig newEmail
+  -- apparently activating the email does not invalidate the activation code
+  -- ideally we would let the activation code expire again (which is too expensive time-wise)
+  -- and then assert that if the email is verified there is no new activation code created
+  -- when the set email function is called again (idempotency)
+  checkSetUserEmail teamOwner emailOwner newEmail 200
```

Comment:
sorry, changed my mind again, i think you were right before :(

```suggestion
  checkLetActivationExpire newEmail
  checkSetUserEmail teamOwner emailOwner newEmail 200
```

if you've had enough just ignore me, but without this line you might as well remove line 163ff altogether, right?

Reply:
I feel indifferent about both approaches. I would suggest to keep it like it is and extend the comment, if that is ok with you?